.github/contributing/adding-tlg.md

```diff
@@ -0,0 +1,64 @@
+# Adding new Tables, Listing and Graphs
+In order to add new TLGs, two things are needed:
+### TLG definition
+Specified in [tlg.yaml](../../inst/shiny/tlg.yaml) file. Entry in that file is responsible for providing some metadata regarding the TLG (like name, descriptions, links), a function reference for creating the resulting TLG and options, taken as arguments by the generating function, that allow for quick and easy definition of widgets to be rendered in the application UI. This then allows the user to customize plots in accordance to their needs.
+
+### `R` function
+The code that creates the actual results.
+
+## Listings
+Currently not implemented.
+
+## Tables
+Currently not implemented
+
+## Graphs
+### yaml
+The `Graph` entry should have the following format:
+ ```yaml
+# unique identifier for given entry
+id:
```

Comment:
suggestion: here and with `optid` consider indicating more clearly that the text is supposed to be replaced by the developer. for example, `<id>`, or `<enter plot id>`, or something else maybe?